packages/cloud-cognitive/src/components/UserProfileImage/_userProfileImage.scss

```diff
@@ -5,16 +5,17 @@
 // LICENSE file in the root directory of this source tree.
 //
 
+@import '@carbon/import-once/scss/import-once';
 @import '../../global/styles/carbon-settings'; // goes before carbon imports as it affects how carbon is used.
 @import '../../global/styles/project-settings';
 @import './colorMap';
 
 $block-class: #{$pkg-prefix}-user-profile-avatar;
-$x-large: $spacing-10; //64px
-$large: $spacing-07; //32px
-$medium: $spacing-06; //24px
-$small: 1.25rem; //20px
-$x-small: $spacing-05; //16x
+$xl: $spacing-10; //64px
+$lg: $spacing-07; //32px
+$md: $spacing-06; //24px
+$sm: 1.25rem; //20px
+$xs: $spacing-05; //16x
```

Comment:
```suggestion
$sizes: (
  xl: $spacing-10, //64px
  lg: $spacing-07, //32px
  md: $spacing-06, //24px
  sm: 1.25rem, //20px
  xs: $spacing-05 //16x
);

@mixin size($size: 'md') {
    $_size: map-get($sizes, $size);
    width: $_size;
    height: $_size;
}
```

Reply:
Added in latest [commit](https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/572/commits/942ee99c00b434a9d3d32b057fcb099b7d78df4d).